app/views/projects/show.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>Projects</h1>
+<h3><%= @project.name %></h3>
+<p>
+  Material: <%= @project.material %><br />
+  Challenge Theme: <%= @project.challenge.theme %><br />
+  Number of Contestants: <%= @project.contestants.count %><br />
```

Comment:
While this is a very small, basic calculation, I would still encapsulate it in a model method in Project (`.count_contestants`) to keep all calculations closer to the db